README.md

```diff
@@ -1,3 +1,3 @@
 # re-former
 
-Waiting for approval
\ No newline at end of file
+Waiting for approval
```

Comment:
Having a descriptive `README.md` is a requirement for *every single project*. Reviewers won't let you pass without one, and it's one of those things that should just come natural to you when creating a project and pushing it to GitHub or another platform. It comes so natural, that sometimes we **reviewers** take it for granted that it is there and **forget** to revise it, as I did in my last review.

I'm letting you pass this time because I didn't raise this on my last review and I don't want to hold you back, but you have to modify this and add a descriptive readme in here. In my defence, it's the first line in additional Microverse requirements:

![image](https://user-images.githubusercontent.com/22228397/65050754-8afb6e00-d92d-11e9-8d0d-bc5527594b76.png)

Please read all instructions carefully every single time, to avoid being delayed any further.

[`REQUIRED`]

Reply:
We had it before, you didn't forget :D. It was just that when we try to fix the master and development branch to be clean before the merge, we somehow delete it on all branches. But it was there before. We have updated it again.

Thanks!